components/Header.tsx

```diff
@@ -260,11 +260,44 @@ const Header: FC<Props> = ({ background }) => {
             })}
           </Popover.Group>
           <div className="hidden md:flex items-center justify-end md:flex-1 lg:w-0">
-            {socialMedia.map(item => (
-              <Link key={item.href} href={item.href} className="mx-2">
-                <item.icon className="h-7 w-7 text-brand-red hover:text-white transition-all cursor-pointer" />
-              </Link>
-            ))}
+            {socialMedia.map(item => {
```

Comment:
Looks good. Can we pull the logic out to either a new file or outside of the JSX?